src/components/sbb-link-list/sbb-link-list.scss

```diff
@@ -24,9 +24,47 @@ sbb-title {
   @include link-list--horizontal;
 }
 
-.link-list--horizontal-from-large {
-  @include mq($from: large) {
-    @include link-list--horizontal;
+.link-list--horizontal-from {
```

Comment:
We prefer to avoid using `&-...`: https://github.com/lyne-design-system/lyne-components/blob/master/CODING_STANDARDS.md#avoid-scss--rule-concatenation
I'm aware that this is nitpicky...
On the other hand it might make sense to use [@each](https://sass-lang.com/documentation/at-rules/control/each) here with `.link-list--horizontal-from-#{breakpoint}`.
What do you think?

Reply:
I like!